shimmingtoolbox/coils/siemens_basis.py

```diff
@@ -12,6 +12,86 @@
 GYROMAGNETIC_RATIO = 42.576
 
 
+def reorder_to_siemens(spher_harm):
```

Comment:
If they are just used here would not need to be called elsewhere, I would call them _reorder_to_siemens and _get_scaling_factor.